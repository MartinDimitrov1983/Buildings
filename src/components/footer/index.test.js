import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import Footer from './index';

test('Snapshot Footer', () => {
    const { asFragment } = render(
        <BrowserRouter>
            <Footer />
        </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
});
