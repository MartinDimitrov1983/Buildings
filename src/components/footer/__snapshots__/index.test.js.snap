// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Footer 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <div>
        <a
          class="link"
          href="/"
        >
          HomePage
        </a>
        <a
          class="link"
          href="/details"
        >
          Details
        </a>
      </div>
      <p
        class="text"
      >
        The Building App
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
